arm-apimanagement/2016-07-07/swagger/apimanagement.json

```diff
@@ -6321,6 +6597,150 @@
         "type"
       ],
       "description": "Parameters supplied to the Update Logger operation."
+    },    
+    "BackendGetContract": {
```

Comment:
Please revisit names and inheritence model!
`BackendPaged` is a collection of `BackendGetContract`s?
`BackendGetContract` is a subclass of `BackendCreateParameters`?
These relationships are not intuitive to me.